canonical/canonical.com
# feat: Migrate microstack.run to canonical.com/microstack WD-16625

## Done

- Migrated microstack.run to canonical.com/microstack

## QA

- Visit https://canonical-com-1438.demos.haus/microstack 
  - Be sure to test on mobile, tablet and desktop screen sizes
- Confirm the page is identical to https://microstack.run (other than vanilla depreciations like the suru strip).
- Go to https://canonical-com-1438.demos.haus/microstack/docs and confirm the page is loading relevant documentation.

## Issue / Card

Fixes [WD-16625](https://warthogs.atlassian.net/browse/WD-16625)


[WD-16625]: https://warthogs.atlassian.net/browse/WD-16625?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
